src/main/java/com/adyen/model/checkout/DefaultPaymentMethodDetails.java

```diff
@@ -60,6 +60,10 @@
     private String sepaOwnerName;
     @SerializedName("sepa.ibanNumber")
     private String sepaIbanNumber;
+    @SerializedName("additionalData.applepay.token")
```

Comment:
```suggestion
    @SerializedName("applepay.token")
```

Could you remove the "additionalData." suffix from the key for the token? Following the example shown here: 
https://docs.adyen.com/payment-methods/apple-pay/api-only#apple-pay-api